.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
What is this file for?

Reply:
I have no clue about why these two folders appeared in my branch.....
Deleted

Reply:
It's in the development branch; maybe it's created by Gsy. She used the  Codesandbox for development. 
You may need to delete them in the development branch.